modules/reference/pages/rpk/rpk-connect/rpk-connect.adoc

```diff
@@ -0,0 +1,85 @@
+= rpk connect
+
+A stream processor for mundane tasks.
+
+Either run Redpanda Connect as a stream processor or choose a command:
+
+Available commands:
+
+- xref:reference:rpk/rpk-connect/rpk-connect-echo.adoc[] - Parse a config file and echo back a normalized version.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-lint.adoc[] - Parse Redpanda Connect configs and report any linting errors.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-streams.adoc[] - Run Redpanda Connect in streams mode.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-list.adoc[] - List all Redpanda Connect component types.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-run.adoc[] - Run Redpanda Connect in normal mode against a specified config files.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-create.adoc[] - Create a new Redpanda Connect config.
+
+- xref:reference:rpk/rpk-connect/rpk-connect-test.adoc[] - Execute Redpanda Connect unit tests.
+
+- `rpk connect template` - Interact and generate Redpanda Connect templates.
+
+- `rpk connect blobl` - Execute a Redpanda Connect mapping on documents consumed via stdin.
+
+- `rpk connect studio` - Interact with Benthos studio (https://studio.benthos.dev).
+
+- `rpk connect help` - Shows a list of commands or help for one command.
+
+See https://docs.redpanda.com/redpanda-connect
```

Comment:
Should this be xref:redpanda-connect:ROOT:about.adoc[] ?

Reply:
yup, changing those now